pkgs/games/osu-lazer/bin.nix

```diff
@@ -63,8 +64,11 @@ else appimageTools.wrapType2 {
     let
       contents = appimageTools.extract { inherit pname version src; };
     in
+    builtins.readFile "${makeWrapper}/nix-support/setup-hook" +
```

Comment:
I wonder if there's a better way to include setup hooks with `appimageTools`

Reply:
If `nativeBuildInputs` is not working, I think it would be better to source (e.g.: `. ${makeWrapper}/nix-support/setup-hook`) instead of using `builtins.readFile`.

Reply:
At first I tried putting it in `nativeBuildInputs` too, but it couldn't find `makeWrapper` when executing the install script, sourcing it looks much better, thanks!